### ISSUE TITLE
First participant audio publish issue

### ISSUE THREAD

**Issue by crypt3dC4rb0n** [livekit/client-sdk-unity-web#22]
I was able to setup the project and take a WebGL build in Unity without any issues but came across a bug in publishing the audio (I'm only trying to publish the audio tracks, video is disabled). The issue seems to be only affecting the first participants in a room where in the first participant in the room can hear the audio of all other participants, but other participants are not able to hear the audio of the first participant. I've tested the following scenarios:

- 2 users - 1st user can hear audio from 2nd user, but 2nd user cannot hear audio from 1st user
- 3 users - 1st user can hear audio from 2nd & 3rd user, 2nd & 3rd user can't hear audio from 1st user but can hear each other's audio.
- 3 users and 1st user leaves - When the 3 users were in the room, the observation was same as above. Let's say the users were abc, efg & xyz in their order of joining, after abc leaves the room, xyz can't hear the audio from efg but efg can hear the audio from xyz.

My initial assumption was that maybe the first participant doesn't have publishing permissions or maybe the SDP lines were set to recvonly but all of these seems to be set correctly. All participants are having the same permissions and SDP lines were also set to sendrecv. 

I've tried the similar implementation using JS SDK as well and everything was working well without any issues. This issue seems to be only happening for Unity WebGL builds.  

Tested browsers
- Chrome
- Firefox
- Brave
- Edge
- Safari

**Comment by theomonnom** [comment]
Hey @crypt3dC4rb0n, thanks for reporting the issue.
Which version of Unity are you using?
Are you using our unity demo, or have you created a new project? (so I know if I have to dig into the demo repo)

**Comment by crypt3dC4rb0n** [comment]
Hey, thanks for the quick response. I've tested with the following unity versions

- 2022.1.18f1
- 2021.3.11f1

I tried with your demo as well as a new project but had the same issue in both. I also tried updating the livekit-js sdk version from 1.3.0 to 1.4.2 but it didn't resolve the issue.

Note: I only changed the token fetch part in your code, used UnityWebRequest for fetching the token from a server deployed REST API service.
